src/test/scenarios/requestDeployments.test.js

```diff
@@ -15,8 +20,25 @@ describe("request deployments", function () {
     var nockResponse = [{
       id: "deployment-1",
       affectedApps: ["app1", "app2"],
-      currentActions: ["ScaleApplication"]
-    }, {}, {}];
+      currentActions: [{action: "ScaleApplication"}]
+    }, {
+      id: "deployment-2",
+      affectedApps: ["/app1"],
+      currentActions: [{
+        action: "ScaleApplication",
+        app: "/app1",
+        readinessCheckResults: [{
+          lastResponse: {
+            body: "{\"errors\":[],\"phases\":[{\"blocks\":[{\"has_decision_point\":true,\"id\":\"block1\",\"message\":\"block1\",\"name\":\"block1\",\"status\":\"Waiting\"},{\"has_decision_point\":true,\"id\":\"block2\",\"message\":\"block2\",\"name\":\"block2\",\"status\":\"Pending\"},{\"has_decision_point\":true,\"id\":\"block3\",\"message\":\"block3\",\"name\":\"block3\",\"status\":\"Pending\"}],\"id\":\"phase\",\"name\":\"phase\",\"status\":\"Waiting\"},{\"blocks\":[{\"has_decision_point\":false,\"id\":\"block4\",\"message\":\"block4\",\"name\":\"block4\",\"status\":\"Pending\"},{\"has_decision_point\":false,\"id\":\"block5\",\"message\":\"block5\",\"name\":\"block5\",\"status\":\"Pending\"},{\"has_decision_point\":true,\"id\":\"block99\",\"message\":\"block99\",\"name\":\"block99\",\"status\":\"Pending\"}],\"id\":\"phase1\",\"name\":\"phase1\",\"status\":\"Pending\"}],\"status\":\"Waiting\"}",
```

Comment:
Shouldn't be a negative comment..
But yeah, that makes sense with the `.status`! :)